src/pages/Troubleshooting.vue

```diff
@@ -139,6 +143,7 @@ export default {
       ipcRenderer.send('troubleshooting/show-logs');
     },
     updateDebug(value) {
+      this.$data.isDebugging = value;
```

Comment:
```suggestion
      this.$data.isDebugging = runInDebugMode(value);
```
(I realize that it's read only in the relevant case, but it's better to be clear.)

Reply:
Well, if you're calling `updateDebug` it means the checkbox isn't disabled, which means that the env var isn't set. But in the future we might bring in another field, so ok...